app/Utils/index.js

```diff
@@ -1,2 +1,9 @@
+import { Client } from 'bugsnag-react-native';
+
 const APLHABETS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
-export { APLHABETS };
\ No newline at end of file
+const bugsnag = new Client("e1b5d94f16186f8c6a2169882998ebda");
```

Comment:
The API key should be used from .env and not committed to GitHub.

Reply:
Fixed.